### DISCUSSION TITLE
Installation Issue

### DISCUSSION THREAD

**Discussion by yj9k5789** [coin-or/Bonmin#33] [Q&A]
I tried to install BONMIN with Cygwin. I have finished the configuration part. It shows "Main configurations of Bonmin sucessful". But when I type "make", it says no such file or directory. Here is the log:

[log_file.txt](https://github.com/coin-or/Bonmin/files/9000707/log_file.txt)

How can I install BONMIN successfully?

**Comment by svigerske** [comment]
I guess this just means that you do not have an executable called "make" in your PATH. If `gmake` doesn't work either, then use Cygwins package manager to install make (or gmake).

**Reply by yj9k5789** [reply]
Thank you for replying. The suggestion works. However there is a new problem. After I run "make install", there is no "bin" folder shows up...

**Reply by svigerske** [reply]
... an executable is only build if you build the AMPL interface. Maybe you didn't do that.

**Reply by yj9k5789** [reply]
I use pyomo as my interface. I thought I don't need AMPL...

**Reply by svigerske** [reply]
Pyomo wants to use Bonmin's AMPL interface, as far as I remember.

**Reply by yj9k5789** [reply]
How do I build AMPL interface? Should I follow this
https://github.com/coin-or-tools/ThirdParty-ASL

**Reply by svigerske** [reply]
You probably just need to run `get.ASL` in `C:/Bonmin-1.8.8/ThirdParty/ASL` and then configure and build again.
You'll want to get also Mumps, at least, otherwise Ipopt (and thus Bonmin) won't run.

**Reply by yj9k5789** [reply]
Should I get Mumps first before I configure and build Bonmin again?

**Reply by svigerske** [reply]
Yes.
